qualtran/_infra/data_types.py

```diff
@@ -190,6 +232,24 @@ def __attrs_post_init__(self):
     def num_qubits(self):
         return self.bitsize
 
+    def to_bits(self, x: int) -> Iterable[int]:
+        """Yields individual bits corresponding to binary representation of x"""
+        from qualtran.cirq_interop.bit_tools import iter_bits
```

Comment:
We should probably move this to somewhere outside of cirq_interop. There should probably be some consolidation with https://github.com/quantumlib/Qualtran/blob/main/qualtran/simulation/classical_sim.py#L40

Reply:
I've removed dependence on `cirq_interop` for now so that all bit manipulation code is consolidated in this file. Let's do the consolidation in follow-up PRs?

Reply:
I opened #811 to track